gym/f110_gym/envs/track/track.py

```diff
@@ -125,6 +130,12 @@ def from_track_name(track: str):
             occupancy_map[occupancy_map <= 128] = 0.0
             occupancy_map[occupancy_map > 128] = 255.0
 
+            # if exists, load edt
+            if (track_dir / f"{track}_map.npy").exists():
+                edt = np.load(track_dir / f"{track}_map.npy")
+            else:
+                edt = None
+
```

Comment:
I would suggest to 
- integrate the creation of the EDT here
- check existence of EDT asset and its last modification time to ensure the map image has not been changed